### PR TITLE
Rc/main stufe 4

### PR DESCRIPTION
**Pull request by f-peverali**
# Stufen Upgrade

Version: 4.0.0-rc3

Diesen PR NICHT (nach mains-stufe-3) mergen !!!

Date: 25.6.2024

## Description

This is a Pullreuqest that requires an increase in the Version number. Therefore, multiple outside-github, related Task have to be performed and checked.

All jobs with an `x` in the boxes were performed to the best of knowledge.

## Pre-Merge Activities

- [ ] This PR refers to a versioned Branch with a name and a version number in the form of N.n.n, e.g. "TC_3.2.1".
- [x] This PR has a clean meaningful commit history. Minor commits or commits without description have been squashed, at the latest now.
- [x] The _./github/workflows/main.yml_ refers to the correct Firely Terminal and SUSHI Version. 
- [x] By running the _Release_Publish.py_ script, release version and date was updated accordingly. The script ran without errors.
- [x] Eventually, increase the dependency of to newer Basis Modul (package and sushi-config)
- [x] New Release Notes were created, alined to the commit history and cleaned. In Github, go to 
  - [x]  _-> Releases_ then _-> Draft a new release_ with the _Modul Name and Version_, then
  - [x] _-> Target the main-Branch_ and _->enter a new Tag according to the Version_, then click.
  - [x] Click _-> Generate Release notes_ , _->Adjust them if necessary_ and _-> Copy/Paste the Details in the RealeaseNotes.md_ of the very Branch you want to merge.
  - [x] Finally _-> Save as Draft_

## Merge and Publishing

- [x] With the updated Version, Dates, and Release Notes (as described above) with the last committ into the Branch you want to merge.
- [x] In GitHub _-> Actions_ the _->CI (FHIR Validation)_ workflow terminates successfully.
- [x] Add the Approve / the PR gets positively reviewed by a colleague.
- [x] Merge (without squash) the PR, delete the Branch.

## Post-Merge Activities

- [x] Go to the corresponding SIMPLIFIER Project and _-> Github -> Reimport_ the project.
- [ ] Go to the corresponding SIMPLIFIER Project and _-> Packages -> Expand the Dropdown for Create -> Create new package_ for the project.
  - [x] With the corresponding version number, and
  - [x] The Release notes (from above) and a compare-link to the previous Release.
  - [x] Unlist the old package by _-> clicking on the old package_, _-> go to Administration_ and _-> click on Unlist_
- [x] Publish the previously drafted Release, including version number, on GitHub.
- [x] Provide / Archive the IG in the corresponding _gh-pages_ branch of the GitHub project.
  - [x] Checkout the Branch (no need to merge it later).
  - [x] Export from Simplifier via _-> Guides -> Expand the Modul ... -> Export_
  - [x] Unpack the zip, remove the packages folder (because its kinda big), and move everything else to a (version corresponding) new folder in the branch folder structure.
  - [x] Update the file index.html and check rendering
  - [x] commit the branch.
- [x] If ISiK Basismodul was updated all depending Modules should be updated  with a renewed dependency to the incremented Basismodul version - possibly including and closing technical corrections

### CODE REVIEW

File: ImplementationGuide/markdown/Versicherungsverhaeltnis/Versicherungsverhaeltnis_Interaktionen.md [hunk 1]
```diff
@@ -14,13 +14,6 @@ Folgende Suchparameter sind für das Bestätigungsverfahren relevant, auch in Ko
 
    Anwendungshinweise: Weitere Informationen zur Suche nach "_id" finden sich in der [FHIR-Basisspezifikation - Abschnitt "Parameters for all resources"](https://hl7.org/fhir/R4/search.html#all).
 
-2. Der Suchparameter "identifier" zur Suche nach einer Krankenversicherten-ID MUSS unterstützt werden:
```
Comment (f-peverali) [inline ImplementationGuide/markdown/Versicherungsverhaeltnis/Versicherungsverhaeltnis_Interaktionen.md:17]: question: ist das so beabsichtigt gewesen? @alexzautke @patrick-werner  @simoneOnFhir  ?

Comment (alexzautke) [reply ImplementationGuide/markdown/Versicherungsverhaeltnis/Versicherungsverhaeltnis_Interaktionen.md:17]: Nicht das mir bewusst ist

Comment (alexzautke) [reply ImplementationGuide/markdown/Versicherungsverhaeltnis/Versicherungsverhaeltnis_Interaktionen.md:17]: Ja, ist so richtig

Comment (alexzautke) [reply ImplementationGuide/markdown/Versicherungsverhaeltnis/Versicherungsverhaeltnis_Interaktionen.md:17]: Das kommt hier her: https://github.com/gematik/spec-ISiK-Basismodul/pull/362

**Pull request closed without merging**